src/wp-includes/class-wp-admin-bar.php

```diff
@@ -67,7 +67,7 @@ public function initialize() {
 			$header_callback = '_admin_bar_bump_cb';
 		}
 
-		add_action( 'wp_head', $header_callback );
+		add_action( 'wp_head', $header_callback, 7 );
```

Comment:
This priority change will break some plugins, including AMP which tris to remove this action: https://wpdirectory.net/search/01H9VK0AMN4BRDJ0K2Z8WY63GS

Reply:
https://github.com/WordPress/wordpress-develop/pull/4824/files#r1321337020